contributors/design-proposals/propagation.md

```diff
@@ -58,6 +58,21 @@ with shared flag, so that all bind mounts are propagated across mount namespaces
 and references to network namespaces persist.
 
 
+## Concerns about "shared" propagation mode
+
+We noted some issues causing unsteadiness when mounts are specified "shared".
+
+1. When using systemd to start docker, MountFlags must set shared instead of slave
+by default. (see https://github.com/docker/docker/issues/19625)
+
+1. As kubelet might see different mount tree with docker, above issue is difficult
+to detect by kubelet.
+
+1. Shared bindmount may produce explosive growth of shared bindmounts in various
```

Comment:
That's the worse case scenario, but it also affects other benign-looking cases, see https://github.com/kubernetes/community/pull/193#issuecomment-273863723 in this same thread.